test/strings/strchr_fail.c

```diff
@@ -0,0 +1,12 @@
+#include <string.h>
+#include "smack.h"
+
+// @expect error
+
+int main(void) {
+  char *notAWebAddress = "word";
+  char *dotToEnd = strchr(notAWebAddress,'.');
+
+  assert(dotToEnd != 0); // or assert(dotToEnd);
```

Comment:
Either change the assertion (and remove the comment) or just remove the comment after it.